query-model/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/querymodel/jpql/expression/Expressions.kt

```diff
@@ -300,6 +301,14 @@ object Expressions {
         return JpqlRound(value, scale)
     }
 
+    /**
+     * Creates an expression that represents the modulo of values.
+     */
+    @SinceJdsl("3.4.0")
+    fun <T : Int> mod(value1: Expression<T>, value2: Expression<Int>): Expression<T> {
```

Comment:
Since the Expression interface has its T type as out, it doesn't seem necessary to declare generic in the mod function. 

```suggestion
    fun mod(value1: Expression<Int>, value2: Expression<Int>): Expression<Int> {
```

Reply:
Reflected now.